File: app/src/main/java/ru/andrey/cleandictionary/data/network/translation/YandexApi.java
package ru.andrey.cleandictionary.data.network.translation;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;
import ru.andrey.cleandictionary.data.network.model.TranslationResponseModel;

public interface YandexApi {
    @GET("/api/v1.5/tr.json/translate")
    Call<TranslationResponseModel> getData(@Query("key") String apiKey,
                                           @Query("text") String word,
                                           @Query("lang") String langCodes);
}

File: data/src/main/java/ru/andrey/data/api/YandexApi.kt
package ru.andrey.data.api

import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Query

interface YandexApi {

    @GET("/api/v1.5/tr.json/translate")
    fun getTranslation(@Query("text") word: String,
                       @Query("lang") langCodes: String): Single<TranslationResponseModel>
}
